README.md

```diff
@@ -1 +1,14 @@
-A base repo for all microservices
+# A base repo for all microservices
+
+### Running locally using docker compose
+#### Option 1 - All services
+```sh
+export PROFILE=dev
+docker-compose up -d
+```
+
+#### Option 2 - All services in local mode, without service discovery nor config
```

Comment:
This is the default, so maybe list it first, and then mention you can define a different value to pass to the apps.